apps/storybook/src/Slider.stories.tsx

```diff
@@ -20,19 +20,36 @@ export default {
   args: {
     thumbMode: 'inhibit-crossing',
     trackDisplayMode: 'auto',
+    orientation: 'horizontal',
   },
 } as Meta<SliderProps>;
 
+const sliderWrapper = (element: ReactNode, args: any) => {
+  return (
+    // TODO: Is there a way to avoid the hardcoded height of 400px? height of 100% does not work
+    <div
+      className='slider-wrapper'
+      style={{
+        ...(args.orientation == 'vertical'
+          ? { height: '400px', width: 'fit-content' }
+          : {}),
+      }}
+    >
+      {element}
+    </div>
+  );
+};
```

Comment:
I have a `sliderWrapper()` method that wraps the slider in container of `height: 400px` when the orientation is vertical. I have it hardcoded in the example to 400px. Is this okay? I tried using a height of window.innerHeight, but it doesn't refresh/rerender on dimension changes (of the storybook iframe or even the browser). `height: 100%` also doesn't work. Can you give any ideas to avoid the hardcoded height? Or is the hardcoding fine?

Reply:
It is not bad to hardcode height in stories. Check `SideNavigation.stories.tsx` how to wrap stories properly by using `decorators` 
![image](https://user-images.githubusercontent.com/36186912/181280696-f48cd0b2-25bb-4aa9-a2d1-b0c3286a4715.png)

Reply:
**[DONE]** Thanks for the idea!

Reply:
Have you forgot to push? 😁

Reply:
Haha, I've done that mistake before! But this time, I think I did push the code(commit hash: `5bc5bcf7e60b7eb9de27d403d44fc9e6e9d56695`). Are you sure you referred to the latest version or looked at an older version of this code?

Reply:
But you are not using decorators that's why I mistaken it.

Reply:
Ohh, yes, I made this change near line 34 of `Slider.stories.tsx`. Is this my change alright, or I have to make some change to use decorators?

Reply:
Better use decorators as we are using them mostly for this purpose. I also think you don't really need to check whether it is vertical or not, just set height on decorator/wrapper as it won't really cause any problems.

Reply:
**[DONE]** I looked at the documentation, wow, decorators are very useful. I replaced `sliderWrapper()` with using decorators